docs/references/rest-apis/rest-cloudconnection-api.md

```diff
@@ -0,0 +1,726 @@
+        },
+        {
+            "cloudConnectionFactoryPid": "org.eclipse.kura.cloudconnection.raw.mqtt.cloud.RawMqttCloudEndpoint",
+            "defaultCloudEndpointPid": "org.eclipse.kura.cloudconnection.raw.mqtt.CloudEndpoint",
+            "cloudEndpointPidRegex": "^org.eclipse.kura.cloudconnection.raw.mqtt.CloudEndpoint(\\-[a-zA-Z0-9]+)?$"
+        },
+        {
+            "cloudConnectionFactoryPid": "org.eclipse.kura.camel.cloud.factory.CamelFactory",
+            "defaultCloudEndpointPid": "org.eclipse.kura.camel.cloud.factory.CamelFactory",
+            "cloudEndpointPidRegex": "^org.eclipse.kura.camel.cloud.factory.CamelFactory(\\-[a-zA-Z0-9]+)?$"
+        }
+    ],
+    "pubSubFactories": [
+        {
+            "factoryPid": "org.eclipse.kura.cloudconnection.raw.mqtt.subscriber.RawMqttSubscriber",
+            "cloudConnectionFactoryPid": "org.eclipse.kura.cloudconnection.raw.mqtt.cloud.RawMqttCloudEndpoint"
+        },
+        {
+            "factoryPid": "org.eclipse.kura.cloudconnection.eclipseiot.mqtt.CloudPublisher",
+            "cloudConnectionFactoryPid": "org.eclipse.kura.cloudconnection.eclipseiot.mqtt.ConnectionManager",
+            "defaultPid": "org.eclipse.kura.cloudconnection.eclipseiot.mqtt.CloudPublisher",
+            "defaultPidRegex": "^org.eclipse.kura.cloudconnection.eclipseiot.mqtt.CloudPublisher(\\-[a-zA-Z0-9]+)?$"
+        },
+        {
+            "factoryPid": "org.eclipse.kura.cloud.subscriber.CloudSubscriber",
+            "cloudConnectionFactoryPid": "org.eclipse.kura.cloud.CloudService"
+        },
+        {
+            "factoryPid": "org.eclipse.kura.cloud.publisher.CloudPublisher",
+            "cloudConnectionFactoryPid": "org.eclipse.kura.cloud.CloudService"
+        },
+        {
+            "factoryPid": "org.eclipse.kura.cloudconnection.raw.mqtt.publisher.RawMqttPublisher",
+            "cloudConnectionFactoryPid": "org.eclipse.kura.cloudconnection.raw.mqtt.cloud.RawMqttCloudEndpoint"
+        },
+        {
+            "factoryPid": "org.eclipse.kura.event.publisher.EventPublisher",
+            "cloudConnectionFactoryPid": "org.eclipse.kura.cloud.CloudService"
+        }
+    ]
+}
+```
+- 500 Internal Server error
+
+## POST methods
+
+#### Get StackComponents Pids
+
+- Description: This method retrieves all all PIDs of Component instances that make up the stack for a specific [CloudEndpoint](https://download.eclipse.org/kura/docs/api/5.4.0/apidocs/org/eclipse/kura/cloudconnection/CloudEndpoint.html), including its [CloudService](https://download.eclipse.org/kura/docs/api/5.4.0/apidocs/org/eclipse/kura/cloud/CloudService.html), [DataService](https://download.eclipse.org/kura/docs/api/5.4.0/apidocs/org/eclipse/kura/data/DataService.html), [DataTransportService](https://download.eclipse.org/kura/docs/api/5.4.0/apidocs/org/eclipse/kura/data/DataTransportService.html)
```

Comment:
It is not given that the cloud stack is composed of CloudService, DataService and DataTransportService.